theme.toml

```diff
@@ -49,6 +49,9 @@ label = "honey"
 "ui.linenr.selected" = { fg = "lilac" }
 "ui.statusline" = { fg = "lilac", bg = "revolver" }
 "ui.statusline.inactive" = { fg = "lavender", bg = "revolver" }
+"ui.statusline.normal" = { fg = "midnight", bg = "lilac" }
+"ui.statusline.insert" = { fg = "midnight", bg = "mint" }
+"ui.statusline.select" = { fg = "midnight", bg = "chamois" }
```

Comment:
I'd use either lavender or the delta color here

Reply:
@archseer for which mode, `fg` or `bg` ?

Reply:
`bg` of course

Reply:
I tried your suggestions, but its not looking very nice. Are you sure about these. I am fine to add these but just wanted to make sure before i commit because i ran the build its not looking great. But I guess these things are subjective.